web/themes/custom/pineapple/templates/content/node--article--full.html.twig

```diff
@@ -106,4 +106,20 @@
         } %}
       </div>
     </div>
+  {{ content.field_a_article_comments }}
+  {% set comment_count = node.field_a_article_comments.comment_count %}
+  {% if comment_count > 10 and not logged_in %}
+    <div class="forbidden-comments">
+      {% include "@components/_link.twig" with {
+        link_text: 'Log in',
+        link_url: '/user/login'
+      } %}
+      <p> or</p>
+      {% include "@components/_link.twig" with {
+        link_text: ' register',
```

Comment:
Why is there a space before register?

Reply:
Because the links and the text didn't have any space between each other without it but then I added padding so I can take it out

Reply:
And done.